SAP2000_Adapter/CRUD/Read/Loads.cs

```diff
@@ -431,10 +431,10 @@ private List<ILoad> ReadBarVaryingDistributedLoad(List<string> ids = null)
                             case 1:
                                 loads.Add(new BarVaryingDistributedLoad()
                                 {
-                                    DistanceFromA = dist1[i],
-                                    ForceA = forceA,
-                                    DistanceFromB = dist2[i],
-                                    ForceB = forceB,
+                                    StartPosition = dist1[i],
+                                    ForceAtStart = forceA,
+                                    EndPosition = dist2[i],
+                                    ForceAtEnd = forceB,
```

Comment:
EndPosition always reports absolute value (meters) when pulled from SAP, but does not set RelativePositions = false (seems to be true by default). Therefore if distances are pushed as relative values they won't match the pulled values, and the Relative flag won't give the user a heads up.

Reply:
Great catch @jean-lucjacksonBH ! Have updated!